apps/core/lib/core/snitch/data/schema/order.ex

```diff
@@ -22,7 +22,7 @@ defmodule Core.Snitch.Data.Schema.Order do
     # field :shipping
     # field :payment
```

Comment:
commented fields?

Reply:
They haven't been added to the migrations. That's why they are commented.